src/Directory.Packages.props

```diff
@@ -4,20 +4,20 @@
   </PropertyGroup>
   <ItemGroup>
```

Comment:
https://github.com/Azure/serilog-sinks-azuredataexplorer/blob/main/src/Directory.Build.props

Please change version in this file

Reply:
Same question here.

Reply:
Similar answer here, since you will release a new version, the version has to be bumped ?